docs/application/dotnet/samples/multimedia/overview.md

```diff
@@ -98,11 +99,18 @@
 			<p>This sample application demonstrates how to retrieve and set the device volume levels.</p>
 			</td>
 		</tr>
+		<tr>
+			<td><img alt="" height="180" src="media/wvolumecontroller.png" width="180"/></td>
+			<td>
+			<p><strong>(W) VolumeController</strong> [In progress]</p>
+			<p>This sample application demonstrates how to retrieve and set device's volume levels.</p>
```

Comment:
To maintain consistency, can we write as:
"set the device volume levels."

Reply:
done